src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -28,47 +38,94 @@ const mhvBaseUrl = () => {
   return `https://${mhvSubdomain}.myhealth.va.gov`;
 };
 
-export const continueUrl = appId => {
+export const mhvToolName = appId => {
   switch (appId) {
-    case frontendApps.GI_BILL_BENEFITS:
-      return '/education/gi-bill/post-9-11/ch-33-benefit/status';
+    case frontendApps.HEALTH_RECORDS:
+      return 'VA Blue Button';
 
-    case frontendApps.DISABILITY_BENEFITS:
-      return '/disability-benefits/apply/form-526-disability-claim/veteran-information';
+    case frontendApps.RX:
+      return 'Prescription Refill and Tracking';
 
-    case frontendApps.LETTERS:
-      return '/records/download-va-letters/letters';
+    case frontendApps.MESSAGING:
+      return 'Secure Messaging';
 
-    case frontendApps.VETERAN_ID_CARD:
-      return 'https://vicbdc.ppd.vba.va.gov/VIC';
+    case frontendApps.LAB_AND_TEST_RESULTS:
+      return 'Lab and Test Results';
 
-    default:
-      return '/';
+    case frontendApps.APPOINTMENTS:
+      return 'VA Appointments';
+
+    default: // Not a recognized health tool.
   }
+
+  return null;
 };
 
-export const redirectUrl = contentUrl => {
-  switch (contentUrl) {
-    case '/health-care/secure-messaging/':
-      return `${mhvBaseUrl()}/mhv-portal-web/secure-messaging`;
+export const toolUrl = appId => {
```

Comment:
Consolidated `redirectUrl` and `continueUrl` into one helper to determine what URL the CTA should lead to.

Reply:
I could totally see us needing to add GA tags in the future, will that be easily supported by this change?

Reply:
Yup, any string manipulation shouldn't be an issue. Could just add another helper to append GA params.